CMakeLists.txt

```diff
@@ -33,8 +33,7 @@ if(ENABLE_FASTCGI)
   list(APPEND THIRD_PARTY_MODULES thrift)
 endif()
 
-# Add bundled libafdt if the system one will not be used
-if(NOT LIBAFDT_LIBRARY)
+if (NOT MSVC)
```

Comment:
It is correct, because we don't support a non-bundled libafdt. See [here](https://github.com/facebook/hhvm/blob/master/CMake/HPHPFindLibs.cmake#L571) where we link directly against the target.